packages/opentelemetry-cloud-trace-exporter/src/transform.ts

```diff
@@ -147,3 +147,35 @@ function valueToAttributeValue(
       return {};
   }
 }
+
+const HTTP_ATTRIBUTE_MAPPING: {[key: string]: string} = {
+  'http.method': '/http/method',
+  'http.url': '/http/url',
+  'http.target': '/http/target',
```

Comment:
not in https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces#get